tests/utils/test_tf_environment.py

```diff
@@ -0,0 +1,11 @@
+import pytest
```

Comment:
can we please have this module in `rasa.utils.tensorflow.test_environment` ? This way it mirrors the structure of the production code.

Reply:
Done.